content/en/agent/faq/log4j_mitigation.md

```diff
@@ -180,6 +180,39 @@ Finally, start the Datadog Agent service to apply the changes.
 
 `Jmxfetch.jar` is included in the AIX agent install bundle, but there is no code in the AIX agent that runs the `jmxfetch` code. If you are not manually starting the `jmxfetch` process, the `jmxfetch.jar` is not used and can be deleted from `/opt/datadog-agent/bin/agent/dist/jmx/jmxfetch.jar`.
 
+### Containerized Ecosystems
+
+If you are running the Datadog Agent as a container (i.e. in Kubernetes, Nomad or vaniall Docker) and use the JMX version (image name ends in `-jmx`), to remove the JndiLookup.class, you will need to build a custom image of the Datadog Agent.
+
+You can use the following Dockerfile to build the image:
+
+```
+FROM gcr.io/datadoghq/agent:7.X.Y-jmx
+
+RUN apt update && apt install zip -y
+
+RUN zip -q -d /opt/datadog-agent/bin/agent/dist/jmx/jmxfetch.jar org/apache/logging/log4j/core/lookup/JndiLookup.class
+
+RUN apt purge zip -y
+```
+
+Note that you will need to replace X and Y in the tag name.
```

Comment:
If you use my proposed argument-based workflow, you could probable axe this entire paragraph.

Do give a test to my proposal to double-check if it actually works 😅 (it should).

Reply:
great suggestiong